modules/getting-started-cli-scaling-app.adoc

```diff
@@ -1,30 +1,28 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-cli.adoc
+// * tutorials/dev-app-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-cli-scaling-app_{context}"]
-= Scaling the application
+= Scaling up the deployment
 
-In Kubernetes, a `Deployment` object defines how an application deploys. In most cases, users use `Pod`, `Service`, `ReplicaSets`, and `Deployment` resources together. In most cases, {product-title} creates the resources for you.
+In Kubernetes, a `Deployment` object defines how an application deploys. In most cases, users use `Pod`, `Service`, `ReplicaSet`, and `Deployment` resources together. In most cases, {product-title} creates the resources for you.
 
-When you deploy the `national-parks-app` image, a deployment resource is created.  In this example, only one `Pod` is deployed.
+When you deploy the `parksmap` image, a deployment resource is created. In this example, only one pod is deployed. You might want to scale up your application to keep up with user demand or to ensure that your application is always running even if one pod is down.
 
-The following procedure scales the `national-parks-image` to use two instances.
+The following procedure scales the `parksmap` deployment to use two instances.
```

Comment:
Consider removing this line since the first line of the Procedure states that you're scaling the deployment to use two instances.